querybook/webapp/__tests__/lib/sql-helper/sql-checker.test.ts

```diff
@@ -0,0 +1,59 @@
+import { getDroppedTables } from 'lib/sql-helper/sql-checker';
+
+describe('getDroppedTables', () => {
+    test('drops tables', () => {
+        const query = `
+        drop table db.table1;
+        drop table db.table2;
+    `;
+        expect(getDroppedTables(query)).toStrictEqual([
+            'db.table1',
+            'db.table2',
+        ]);
+    });
+
+    test('drops tables with if exists', () => {
+        const query = `
+        drop table if exists db.table1;
+        drop table if exists db.table2;
+    `;
+        expect(getDroppedTables(query)).toStrictEqual([
+            'db.table1',
+            'db.table2',
+        ]);
+    });
+
+    test('drops a table and recreated', () => {
+        const query1 = `
+        drop table if exists db.table1;
+        create table db.table1 if not exists;
```

Comment:
https://prestodb.io/docs/current/sql/create-table.html
create table [if not exists]